sequence_transform.c

```diff
@@ -11,10 +11,11 @@
 #include "qmk_wrapper.h"
 #include "st_debug.h"
 #include "sequence_transform.h"
+#include <stdint.h>
 #include "sequence_transform_data.h"
 #include "utils.h"
 
-#ifndef SEQUENCE_TRANSFORM_GENERATOR_VERSION_3
+#ifndef SEQUENCE_TRANSFORM_GENERATOR_VERSION_2_0
```

Comment:
why are we going from 3 to `2_0` ?

Reply:
We can go back to a single number. I figured I would prepend the 2_ there so that it wouldn't conflict if we update the original uint16_t trie generator before merging this